src/runtime/pkg/containerd-shim-v2/create.go

```diff
@@ -10,22 +10,22 @@ package containerdshim
 import (
 	"context"
 	"fmt"
+	containerd_types "github.com/containerd/containerd/api/types"
+	"github.com/containerd/containerd/mount"
+	taskAPI "github.com/containerd/containerd/runtime/v2/task"
+	"github.com/containerd/typeurl"
 	"github.com/kata-containers/kata-containers/src/runtime/pkg/utils"
 	"github.com/kata-containers/kata-containers/src/runtime/virtcontainers/pkg/rootless"
+	"github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/pkg/errors"
```

Comment:
could you seperate golang deps, external project deps and kata internal deps, with a blank line between them? Just so that they look more clear.
E.g., 
```
"context"
"fmt"

"github.com/containerd/containerd/mount"
"github.com/pkg/errors"

"github.com/kata-containers/kata-containers/src/runtime/pkg/utils"
```